src/components/PeopleFilters.tsx

```diff
@@ -1,18 +1,94 @@
-export const PeopleFilters = () => {
+import React from 'react';
+import cn from 'classnames';
+import { NavLink, Link, useSearchParams } from 'react-router-dom';
+import { Filter } from '../utils/filter';
+
+type Props = {
+  filter: Filter;
+  onFilterChange: React.Dispatch<React.SetStateAction<Filter>>;
+  handleQChange: React.ChangeEventHandler<HTMLInputElement>;
+  query: string;
+};
+
+export const PeopleFilters: React.FC<Props> = ({
+  filter,
+  onFilterChange,
+  handleQChange,
```

Comment:
```suggestion
  handleQueryChange,
```
bad name - what is `Q`? Do not use such short forms